### PR TITLE
docs: Remove travis CI build status in appium-doctor

### PR DESCRIPTION
**Pull request by Dor-bl**
## Proposed changes

I've updated the Build status link on appium-doctor.
Since June 15th, 2021, the building on travis-ci.org is ceased. so need to use travis-ci.com from now on.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/doctor/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ appium-doctor
 ===================
 Attempts to diagnose and fix common Node, iOS and Android configuration issues before starting Appium.
 
-[![Build Status](https://travis-ci.org/appium/appium-doctor.svg?branch=master)](https://travis-ci.org/appium/appium-doctor)
+[![Build Status](https://travis-ci.com/appium/appium-doctor.svg?branch=master)](https://app.travis-ci.com/github/appium/appium-doctor)
```
Comment (KazuCocoa) [inline packages/doctor/README.md:5]: We no longer has used Travis, so this PR can be the removal

Comment (Dor-bl) [reply packages/doctor/README.md:5]: So what do you use for build CI?

Comment (KazuCocoa) [reply packages/doctor/README.md:5]: GitHub Actions -> https://github.com/appium/appium/actions/workflows/build.yml

Comment (Dor-bl) [reply packages/doctor/README.md:5]: And I assume there is no specific build for appium doctor. So I should refer to the appium build when im updating the build status for appium doctor.

Comment (KazuCocoa) [reply packages/doctor/README.md:5]: Yes, for current CI configuration. We may be able to configure CI for each package in this repo, but I think current execution time of entire this repo is acceptable, then we can keep simple CI tasks so far

Comment (Dor-bl) [reply packages/doctor/README.md]: I've removed the build status of Travis as discussed.

**Review by KazuCocoa — APPROVED** [review]
thanks

**Pull request merged**